Code/GraphMol/RGroupDecomposition/RGroupDecompData.h

```diff
@@ -472,13 +553,19 @@ struct RGroupDecompData {
           if (heavy_counts < largest_heavy_counts) {
             largest_heavy_counts = heavy_counts;
             best_permutation = tied_permutation;
-          } else if (heavy_counts == largest_heavy_counts) {
-            size_t perm_value = iterator.value();
-            if (perm_value < min_perm_value) {
-              min_perm_value = perm_value;
-              best_permutation = tied_permutation;
-            }
           }
+          // commented out as min_perm_value and perm_value are unsigned
```

Comment:
@bp-kelley  I guess I can't see how perm_value can ever be less than min_perm_value.  Both variables are unsigned  and min_perm_value is initialized to zero.  min_perm_value is only updated when `perm_value < min_perm_value` which can never happen.  So, I think it is dead code.

Reply:
@jones-gareth Yes, I realized that a few days ago while I was working on another issues. There are other bugs in that code; this fix will be part of another PR that I should be able to finalize today or tomorrow as soon as I find the time to write unit tests for it.

Reply:
@ptosco I have just also fixed a couple of bugs in the core rgroup code (see my other comment).  Maybe we could time our changes so that only one of us is working on the code at the same time and we don't have to rebase our branches.